config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,391 @@
+      <property name="tokens"
+        value="COMMA, SEMI, POST_INC, POST_DEC, DOT, ELLIPSIS, METHOD_REF" />
+      <property name="allowLineBreaks" value="true" />
+    </module>
+    <module name="NoWhitespaceAfter">
+      <property name="tokens"
+        value="INC, DEC, UNARY_MINUS, UNARY_PLUS, DOT, LNOT" />
+      <property name="allowLineBreaks" value="true" />
+    </module>
+    <module name="ParenPad" />
+    <module name="OperatorWrap">
+      <property name="option" value="NL" />
+      <property name="tokens"
+        value="BAND, BOR, BSR, BXOR, DIV, EQUAL, GE, GT, LAND, LE, LITERAL_INSTANCEOF, LOR,
+                    LT, MINUS, MOD, NOT_EQUAL, PLUS, QUESTION, SL, SR, STAR, METHOD_REF " />
+    </module>
+    <module name="AnnotationLocation">
+      <property name="id" value="AnnotationLocationMostCases" />
+    </module>
+    <!--<module name="NonEmptyAtclauseDescription"/>-->
+    <module name="JavadocTagContinuationIndentation" />
+    <module name="SummaryJavadoc">
+      <property name="forbiddenSummaryFragments"
+        value="^@return the *|^This method returns |^A [{]@code [a-zA-Z0-9]+[}]( is a )" />
+    </module>
+    <module name="JavadocParagraph" />
+    <module name="AtclauseOrder">
+      <property name="tagOrder" value="@param, @return, @throws, @deprecated" />
+      <property name="target"
+        value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF" />
+    </module>
+    <module name="MethodName">
+      <message key="name.invalidPattern"
+        value="Method name ''{0}'' must match pattern ''{1}''." />
+      <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$" />
+    </module>
+    <module name="SingleLineJavadoc">
+      <property name="ignoreInlineTags" value="false" />
+    </module>
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="expected|ignore" />
+    </module>
+    <module name="CommentsIndentation" />
+
+    <!-- Imports, http://checkstyle.sourceforge.net/config_imports.html -->
+    <module name="AvoidStarImport" />
+    <module name="RedundantImport" />
+    <module name="UnusedImports" />
+    <module name="IllegalImport">
+      <property name="illegalPkgs"
+        value="junit.framework, sun, java.util.stream, java.util.function, java.time, java.nio.file" />
+      <property name="illegalClasses"
+        value="java.util.Optional, java.io.UncheckedIOException, java.util.PrimitiveIterator, java.util.Spliterator, java.util.DoubleSummaryStatistics, java.util.IntSummaryStatistics, java.util.LongSummaryStatistics, java.util.Optional, java.util.OptionalDouble, java.util.OptionalInt, java.util.OptionalLong, java.util.Spliterators, java.util.SplittableRandom, java.util.StringJoiner, java.security.DomainLoadStoreParameter, java.security.cert.PKIXRevocationChecker, java.net.URLPermission, java9.util.stream.IntStream, java9.util.stream.DoubleStream, java9.util.stream.Stream" />
+    </module>
```

Comment:
Ups, sorki, całe to `IllegalImport` jest do wywalenia - zapomniałam, że to tu jest (dziwactwa Androida akurat)